01 - Navigation/01 - Navigate to something you know/Code/JetBrains/Customers/Customer.php

```diff
@@ -20,7 +20,7 @@ function __construct($name, $age)
     }
 
     /**
-     * @param int $age
+     * @param $age int
```

Comment:
I think these changes are incorrect; according to the docs, *type* should be *before* the name of the param; http://www.phpdoc.org/docs/latest/references/phpdoc/tags/param.html